packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -197,8 +197,8 @@ export class Create {
                         break;
 
                     case "blksize":
-                    /* 
-                    *  This is a fix for issue https://github.com/zowe/zowe-cli/issues/1439. 
+                    /*
+                    *  This is a fix for issue https://github.com/zowe/zowe-cli/issues/1439.
                     *
                     */
                         if (isNullOrUndefined(tempOptions.blksize)) {
```

Comment:
`util.isNullOrUndefined` pops up the following deprecated warning in my editor: 
```@deprecated — since v4.0.0 - use value === null || value === undefined instead```
I'm not sure if you'll want to fix this though since there are 17 instances of its use in this file

Reply:
Updated with the latest update.